### PR TITLE
add register_swarm_agent use case

### PR DESCRIPTION
**Pull request by Bonczidai**
Signed-off-by: Bonczidai Levente <levente.bonczidai@hp.com>

### CODE REVIEW

File: content/io/cloudslang/docker/swarm/register_swarm_agent.sl [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+########################################################################################################
+
+namespace: io.cloudslang.docker.swarm
+
+imports:
+  containers: io.cloudslang.docker.containers
+
+flow:
+  name: register_swarm_agent
+  inputs:
```
Comment (meirwah) [inline content/io/cloudslang/docker/swarm/register_swarm_agent.sl]: are you supporting TLS certificates with this operation?

File: test/io/cloudslang/docker/swarm/test_register_swarm_agent.sl [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        navigate:
+          SUCCESS: SUCCESS
+          FAILURE: POST_CLEAR_MACHINE_PROBLEM
+  results:
+    - SUCCESS
+    - FAILURE
+    - PRE_CLEAR_MACHINE_PROBLEM
+    - GET_RUNNING_CONTAINER_IDS_PROBLEM
+    - VERIFY_CONTAINER_IS_RUNNING_PROBLEM
+    - POST_CLEAR_MACHINE_PROBLEM
```
Comment (meirwah) [inline test/io/cloudslang/docker/swarm/test_register_swarm_agent.sl]: I do not think there is much value in this test, you are only  testing here that the container is up and running... a better test maybe to get it up and running, and join an agent to an existing cluster, that way you test both this op and the create manager op, and have a more E2E scenario...

**Comment by Bonczidai** [comment]
migrated to #226

**Comment by orius123** [comment]
@Bonczidai can we delete the branch?

**Comment by Bonczidai** [comment]
yes - I am deleting the branch

**Pull request closed without merging**